app/assets/javascripts/components/CreateTasks.jsx

```diff
@@ -0,0 +1,88 @@
+
+class CreateTasks extends React.Component {
+
+    constructor(props) {
+        super();
+        this.state = {
+            users: [],
+            tasks: []
+        }
+    }
+
+    componentDidMount() {
+        Requester.get('/api/tasks/show?client_id=' + this.props.clientId).then((data) => {
+            this.setState({tasks: data});
+        });
+
+        Requester.get('/api/users/show').then((data) => {
+            this.setState({users: data});
+        });
+    }
+
+    assignTask(taskId, selectId) {
+        select = document.getElementById(selectId);
+        userId = select.value;
```

Comment:
What is the purpose of this `select` variable? I assume we're using this to find the appropriate task element we're modifying. This is not really inline with react idioms/patterns. Talk to me about this when you have time.